src/templates/fr/base.html

```diff
@@ -88,6 +88,18 @@
 {% block author %}Auteur·ice{% endblock %}
 {% block authors %}Auteur·ice·s{% endblock %}
 
+{% block reactions %}Réactions{% endblock %}
+{% block source %}Voir la source{% endblock %}
+
+{% block like %}comme{% endblock %}
```

Comment:
```suggestion
{% block like %}j'aime{% endblock %}
```

Reply:
Going with `j’aime` as this won't be fixed by smart quotes library.

Reply:
Where do I comment? Here or in the new PR?

Reply:
New PR please. Will resolve these comments.